src/pymortests/strategies.py

```diff
@@ -246,6 +246,12 @@ def valid_inds(v, length=None, random_module=None):
         yield list(range(int(len(v)/2)))
         yield list(range(len(v))) * 2
         # TODO what's with the magic number here?
+        # Maybe related to this?
```

Comment:
@sdrave do you have any insight on this?

Reply:
I don't think there is any deeper meaning. It's probably supposed to be an example with repeated indices.